internal/liqonet/tunnel-operator/labelerOperator_test.go

```diff
@@ -50,11 +55,74 @@ var _ = Describe("LabelerOperator", func() {
 				},
 			},
 		}
+		// Declare test service.
+		labelerTestSvc = &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      labelerSvcName,
+				Namespace: labelerReq.Namespace,
+				Labels: map[string]string{
+					podNameLabelKey:      podNameLabelValue,
+					podComponentLabelKey: podComponentLabelValue,
+				},
+			},
+			Spec: corev1.ServiceSpec{
+				Selector: map[string]string{
+					"app": "test",
+				},
+				Ports: []corev1.ServicePort{
+					{
+						Port: 80,
+						TargetPort: intstr.IntOrString{
+							Type:   intstr.Int,
+							IntVal: 80,
+						},
+					},
+				},
+			},
+		}
 		lbc = &LabelerController{
 			PodIP:  labelerCurrentPodIP,
 			Client: k8sClient,
 		}
 	})
+	JustAfterEach(func() {
+		// Remove the existing pod.
+		Eventually(func() error {
+			labelerReq.Name = labelerPodName
+			pod := new(corev1.Pod)
+			err := k8sClient.Get(context.TODO(), labelerReq.NamespacedName, pod)
+			if err != nil && !apierrors.IsNotFound(err) {
+				return err
+			}
+			if apierrors.IsNotFound(err) {
+				return nil
+			}
+			err = k8sClient.Delete(context.TODO(), pod)
+			if err != nil {
+				return err
+			}
+			return nil
```

Comment:
```suggestion
			return k8sClient.Delete(context.TODO(), pod)
```

Nit. Just a bit shorter